firmware.conf

```diff
@@ -49,9 +49,9 @@ url = http://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p198079150
 file = ilo196.bin
 
 [ilo2]
-version = 2.31
-url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p915269625/v142909/CP033889.scexe
-file = ilo2_231.bin
+version = 2.32
+url = http://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p915269625/v144949/CP034429.scexe
```

Comment:
Why switch to `http`? The URL also works with `https`.

Reply:
but all other 100 links in the file are http - either change all to https or stick to http imo

Reply:
Now, that's a good point. @seveas what do you say?